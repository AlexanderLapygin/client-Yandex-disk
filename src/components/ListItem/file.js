import React from 'react';
import icFile from './file.svg';

export const file = (item) => {
  return (
    <div className='d-flex justify-content-between'>
      <div className='item__file_name'>
        <img src={icFile} alt='folder'
             className='file_icon pr-3'/>
        {item.name}
      </div>
      <a href={item.file}>
        <button className='btn btn-outline-success'>
          <i className='fas fa-arrow-down'/>
        </button>
      </a>
    </div>
  );
};
