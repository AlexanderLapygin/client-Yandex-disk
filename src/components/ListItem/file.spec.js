import React from 'react';
import { file } from './file';


describe('file test', () => {
  const props={
    name:'filename',
    file:''
  };
  it('File name',()=>{

    expect(file(props)).toMatchSnapshot()
  })
});
