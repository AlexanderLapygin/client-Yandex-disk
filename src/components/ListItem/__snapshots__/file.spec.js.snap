// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file test File name 1`] = `
<div
  className="d-flex justify-content-between"
>
  <div
    className="item__file_name"
  >
    <img
      alt="folder"
      className="file_icon pr-3"
      src="file.svg"
    />
    filename
  </div>
  <a
    href=""
  >
    <button
      className="btn btn-outline-success"
    >
      <i
        className="fas fa-arrow-down"
      />
    </button>
  </a>
</div>
`;
